src/main/java/org/alfresco/repo/jscript/ScriptNode.java

```diff
@@ -272,6 +278,8 @@ public ScriptNode(NodeRef nodeRef, ServiceRegistry services, Scriptable scope)
         this.fileFolderService = services.getFileFolderService();
         this.retryingTransactionHelper = services.getTransactionService().getRetryingTransactionHelper();
         this.scope = scope;
+        renditionService2 = (RenditionService2)services.getService(RENDITION_SERVICE_2);

```

Comment:
Is it possible to add RenditionsService2 to the `ServiceRegistry` and call `services.getRenditionService2()` ?

Reply:
Just have, but looking up services this way no longer seen as the correct approach.